### DISCUSSION TITLE
Font size on a specific page

### DISCUSSION THREAD

**Discussion by mingyuecall** [marp-team/marp#337] [Q&A] [answered]
Dear friends, could I ask how to change the font size on a specific page? I need to do that coz I need to add a list of bibliography to the last page. Thank you!

**Comment by yhatt** [comment, marked as answer]
Use [`font-size` CSS property](https://developer.mozilla.org/docs/Web/CSS/font-size) with [the scoped style `<style scoped>`](https://marpit.marp.app/theme-css?id=scoped-style).

```markdown
<style scoped>
section {
  font-size: smaller;
}
</style>

- Your contents
- ...
```

**Reply by mingyuecall** [reply]
Thank you so much! It did work!
However, it works better with 'large'.

<style scoped>
section {
  font-size: large;
}
</style>

**Comment by ChrCoello** [comment]
The answer 
```
<style scoped>
section {
  font-size: smaller;
}
</style>
```
is changing the font of both the title and the content of the slide. 
How do one only change the size of the content of a slide, not changing the title ?

**Reply by juan11iguel** [reply]
You need to find the HTML tag for the specific element you want to resize, this is an example:

```
---

![bg left 100%](https://marp.app/favicon.png)

# Fragmented ordered list

1) **Name:** Lola the Micropanda
2) **Job:** CFO at Micropanda Accelerator
3) **IG:** @lolamicropanda

Some text


<style scoped>
  ol {
    font-size: smaller;
  }
  p {
    font-size: larger;
  }
</style>
```

Will produce something like this:
![image](https://github.com/marp-team/marp/assets/48693399/f9f592b6-154e-4400-9fcc-cc9c8a4bbe6f)

**Reply by ChrCoello** [reply]
Nice thanks @juan11iguel
